config/projects.yml

```diff
@@ -22,7 +22,7 @@ admin-search-bundle:
         sonata_admin: ['3']
         ruflin_elastica: ['2']
     1.x:
-      php: ['5.6', '7.0']
+      php: ['7.0']
```

Comment:
7.1 should be supported: https://github.com/sonata-project/SonataAdminSearchBundle/blob/9c3496449e4a6d21a8a196548b378bceb74fa9ad/composer.json#L20